### ISSUE TITLE
Screen readers speak too much information when performing a right click action

### ISSUE THREAD

**Issue by ajborka** [microsoft/vscode#92647]
- vscode version: 1.44.0
- OS Version: Ubuntu 20.04

Steps to Reproduce:

1. Start Orca screen reader.
2. Start vscode.
3. In vscode, open a folder that contains files.
4. If focus does not end up on the files list with focus mode turned on,
 A. Navigate to the files list in explorer.
 B. Press ORCA+A to turn on forms mode.
5. Press the APPLICATIONS key or SHIFT+F10 to bring up the context menu.

Actual results: Orca speaks 'leaving complementary information.' before opening the context menu.
Expected result: Orca speaks 'menu' while the context menu opens. The ideal solution is to have screen readers speak a menu title such as 'file actions menu'. Let's have a discussion on naming context menus.

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by isidorn** [comment]
I do not really know why would orca speak `leaving complementary information`, maybe @joanmarie can help here and give some advice if VS Code is doing something wrong.

When I try the same with Voice Over and NVDA both of them nicely speak `menu`.
As for cusstomising the menu title, not sure if that is easily doable, since currently we set the role of the menu to be `menu` and I believe that makes the screen reader announce `menu`. We can not set a role to a custom string, and reading the menu title via an alert is not the nicest solution.
I am open for ideas.

fyi @sbatten for menus

**Comment by ajborka** [comment]
@isidorn The menu name isn't completely important. We do need to stop the 'leaving complementary information' announcement though.

**Comment by joanmarie** [comment]
Orca presents "leaving" messages for landmarks. But I agree that it shouldn't be presenting that message for VSCode. I'll see what I can do in Orca.

**Comment by joanmarie** [comment]
I cannot make Orca say "leaving complementary content" when in focus mode and getting into the context menu. All I hear is "menu". @ajborka mind providing a full (Orca) debug.out to me?

**Comment by isidorn** [comment]
Closing since there is no progress here.
Once more details are provided I can reopen
